src/averageAges.js

```diff
@@ -37,7 +46,13 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people
+    .filter(person => person.sex === 'f');
+  const womenWithChildren = women
+    .filter(person => people
+      .find(child => child.mother === person.name));
```

Comment:
Same here, you wrote redundant functionality for getting ages 2 times

Reply:
Mykola is right

Reply:
Thanks! Fixed it!